Taskfile.yml

```diff
@@ -66,6 +66,19 @@ tasks:
       - task: generate-rlp
       - task: check-clean-branch
 
+  check-generate-bindings:
+    desc: Checks that generated Go bindings from Solidity contracts are up-to-date (requires a clean git working tree)
+    cmds:
+      - task: setup-contracts
```

Comment:
once we get rid of all the npm stuff entirely, I will clean this up to have a single step, and we can get rid of the cmd below.